### DISCUSSION TITLE
Function design that can be used for reference

### DISCUSSION THREAD

**Discussion by whxuezhe** [Alex313031/Thorium-Win-AVX2#68] [Ideas]
When the two and above audio and video tabs, you can choose the mute other tags, as well as the new Audio control UI, and the tab page status logo. The audio displays the speaker icon, and the video is a simplified player.
Then ask Google Cloud in the new version, along with the unreliable code CI test of AVX and AVX2, how did it progress? Is the release date determined? And hope to increase the CLOSE Web Cache, can you disclose some profile and brief progress instructions in the issue and advance in the Issue; please forgive me to be an acute person; it has been a bit of a long time to stop the update.

**Comment by gz83** [comment]
> When the two and above audio and video tabs, you can choose the mute other tags, as well as the new Audio control UI, and the tab page status logo. The audio displays the speaker icon, and the video is a simplified player. Then ask Google Cloud in the new version, along with the unreliable code CI test of AVX and AVX2, how did it progress? Is the release date determined? And hope to increase the CLOSE Web Cache, can you disclose some profile and brief progress instructions in the issue and advance in the Issue; please forgive me to be an acute person; it has been a bit of a long time to stop the update.

@whxuezhe Thanks for your suggestion, Thorium is built based on the latest branch of Chromium, which allows Thorium to have the latest upstream updates and some feature modifications we have made. We will actively consider and evaluate the feasibility of new feature suggestions.
